lib/cocoapods/target/pod_target.rb

```diff
@@ -343,7 +343,9 @@ def framework_paths
             dsym_source = if dsym_path.exist?
                             "${PODS_ROOT}/#{relative_path_to_sandbox}.dSYM"
                           end
-            FrameworkPaths.new(framework_source, dsym_source)
+            bcsymbolmap_paths = Dir.entries(framework_path.dirname.to_s).find_all { |e| e.end_with?('.bcsymbolmap') } if framework_path.dirname.exist?
```

Comment:
Should these paths include `${PODS_ROOT}/#{some_relative_path}`?

Reply:
Yes, you're right.